### PR TITLE
Sounds added

### PR DESCRIPTION
**Pull request by athompsonScottLogic**

### CODE REVIEW

**Review by mcgill-a — APPROVED** [review]

File: whichone/static/js/game.js [hunk 1]
```diff
@@ -1,6 +1,16 @@
 function stopGame() {
     stopped = true;
+
+    if (!muteSound) {
+        var audio = new Audio('/static/resources/gameover.mp3');
+        audio.volume = 0.3;
+        audio.play();
+    }
+
+    console.log("Game over. Final score: " + userCurrentScore);
```
Comment (connor-settle) [inline whichone/static/js/game.js]: Remove dev logs from source control.

Same below

File: whichone/static/js/main.js [hunk 2]
```diff
@@ -7,6 +7,8 @@ var user = {
     scores: []
 };
 
+var muteSound = false;
+
```
Comment (connor-settle) [inline whichone/static/js/main.js]: Should it be a persisted user setting (i.e. LocalStorage)

File: whichone/static/js/game.js (hunk 1 above)
Comment (athompsonScottLogic) [reply whichone/static/js/game.js]: This got removed and then somehow re-added... sorted.

File: whichone/static/js/main.js (hunk 2 above)
Comment (athompsonScottLogic) [reply whichone/static/js/main.js]: done

**Comment by athompsonScottLogic** [comment]
There was a problem with merging main branch into sounds branch, but that has been fixed and all old and new sounds functionality has been repaired. Mute setting now stores locally

**Review by mcgill-a — CHANGES_REQUESTED** [review]
Still need to improve alignment of the icons and maybe reduce the size of them a bit on small screens

**Comment by PHardySL** [comment]
Looks good, all the sounds work properly and mute button works. Same as Alex's changes.

**Comment by athompsonScottLogic** [comment]
> Still need to improve alignment of the icons and maybe reduce the size of them a bit on small screens

Alignment of icons fixed (I believe). Not resized yet, awaiting understanding resizing through skeleton

**Review by mcgill-a — APPROVED** [review]
All works well with the latest commits

**Pull request merged**